frontend/app-development/features/appContentLibrary/utils/convertOptionListsUsageToCodeListsUsage.ts

```diff
@@ -0,0 +1,16 @@
+import type { OptionListsReferences } from 'app-shared/types/api/OptionsLists';
+import type { CodeListReference } from '@studio/content-library';
+
+export const convertOptionListsUsageToCodeListsUsage = (
```

Comment:
**Nitpicking:**
Since this is a 1:1 mapping operation, maybe we can give the function the more specific verb `map`?

And since the parameter is named `optionListUsages`, maybe we can remove that from the function name, resulting in the shorter name `mapToCodeListsUsages`? I think the variable name will be descriptive enough, given the context it is used in.

```suggestion
export const mapToCodeListsUsages = (
```